src/neural/network_check.cc

```diff
@@ -86,48 +110,117 @@ class CheckComputation : public NetworkComputation {
 
     bool policyAlmostEqual = true;
     for (int i = 0; i < size && policyAlmostEqual; i++) {
-      for (int j = 0; j < NUM_OUTPUT_POLICY; j++) {
+      for (int j = 0; j < kNumOutputPolicies; j++) {
         float v1 = refComp_->GetPVal(i, j);
         float v2 = checkComp_->GetPVal(i, j);
         policyAlmostEqual &= IsAlmostEqual(v1, v2);
       }
     }
 
-    // We could also print on standard output using the uci info string
-    // example:
-    //
-    // info string Check passed
-    // info string ***** error check failed
-    //
-
     if (valueAlmostEqual && policyAlmostEqual) {
-      std::cerr << "Check passed for a batch of " << size << std::endl;
+      std::cerr << "Check passed for a batch of " << size << "." << std::endl;
       return;
     }
 
     if (!valueAlmostEqual && !policyAlmostEqual) {
       std::cerr << "*** ERROR check failed for a batch of " << size
-                << " both value and policy incorrect" << std::endl;
+                << " both value and policy "
```

Comment:
Why string literal split into two (also in lines below)?
Do you need both `\n` and `std::endl` ?  (they do roughly the same)